src/dev/flang/be/jvm/classfile/Expr.java

```diff
@@ -388,7 +728,7 @@ public void code(ClassFile.ByteCodeWriter ba, ClassFile cf)
    *
    * @return Code to produce bytecode for the interface call.
    */
-  public static Expr invokeInterface(String cls, String name, String descr, JavaType rt)
+  public static Expr invokeInterface(String cls, String name, String descr, JavaType rt, int argCount)
```

Comment:
Instead of adding `argCount` as an argument here, try `ClassFileConstants.slotCountForArgs(descr)`.